### DISCUSSION TITLE
Incomplete coverage

### DISCUSSION THREAD

**Discussion by leeberryman** [wmgeolab/geoBoundaries#2235] [General]
@DanRunfola I understand that the list of ISOs you accept up to this point have to be UN member states etc.  As I've started working on my list that I'm leveraging the geoboundaries data for I've found that many island territories and disputed areas don't have coverage in the polygons that lay claim to the disputed area or island territory.  Working on some human geography that has to have shapes for these often forgot about places.  I wondered if you'd be willing to revisit the iso list that you would accept shapes for? ![image](https://user-images.githubusercontent.com/23552439/162429214-62f4047c-b140-4f69-a93f-bc8cf22714fe.jpeg)

**Comment by DanRunfola** [comment]
Already done :).  If you go over to the main geoBoundaries website, you can
see the full list of accepted ISO codes in the download table.  Most of the
territories you note are blank - i.e., we don't have an official boundary,
but the tool accepts the ISO codes.

We've been focused on getting our existing database 100% CC-BY compliant,
but territories are definitely a goal.

**Reply by leeberryman** [reply]
That's great!  So if I do the leg work for submitting these they can be included?

**Comment by DanRunfola** [comment]
Yes, the system is all setup and ready to go for them, just haven't been
prioritized yet.

**Reply by leeberryman** [reply]
@DanRunfola I'm finding it difficult to find images or maps without share alike constraints to digitize. I do have the ability to digitize from sentinel-2-l2a imagery since these are mostly islands. Please see https://scihub.copernicus.eu/twiki/pub/SciHubWebPortal/TermsConditions/Sentinel_Data_Terms_and_Conditions.pdf for the data license for sentinel-2-l2a imagery can you affirm that this license will be acceptable for digitizing the island territories that are missing?

This approach wouldn't work for those ISOs that are not islands, but most of these missing are the island case.

**Comment by DanRunfola** [comment]
Yes, I can confrim that if you digitize from a satellite image that would be considered a derivative work, and thus you woul dbe free to license it however you choose (optimally, CC-BY so we can integrate it here!).

**Reply by leeberryman** [reply]
Thanks @DanRunfola.  I'm assigning these to one of our developers.  Thankful for Geoboundaries and how  you guys have built an ever improving dataset that the community can contribute to.  I asked a while back, but is it possible to buy you a coffee and chat sometime?  Is summer a good time to drop by?

**Comment by DanRunfola** [comment]
@leeberryman Thanks - we're actually just about to hire a developer as well fysa - will be sharing that announcement with the community soon.  Shoot me an email (danr@wm.edu) and we can coordinate coffee sometime!
